### ISSUE TITLE
"Zoom and Pan" video gesture

### ISSUE THREAD

**Issue by SameenAhnaf** [TeamNewPipe/NewPipe#5676]
### Checklist

- [x] I checked, but didn't find any duplicates (open OR closed) of this issue in the repo. 
- [x] I have read the contribution guidelines given at https://github.com/TeamNewPipe/NewPipe/blob/HEAD/.github/CONTRIBUTING.md.
- [x] This issue contains only one feature request. I will open one issue for every feature I want to request.

#### Describe the feature you want

I need to watch a specific part of a video very often. I will be highly grateful to be able to stream videos this way. I think the following gesture for "Zoom and pan" is the best.

Pinch two fingers together in parallel position to pan the video. Pinch two fingers like zooming a photo for zooming in or out the video.

#### How will you/everyone benefit from this feature?

Enjoy specific part of a video without any hassle.

**Comment by mhmdanas** [comment]
Closing as a duplicate of #1406. You can write a comment there about the pan gesture.
